### ISSUE TITLE
Setting show fanart to seasons

### ISSUE THREAD

**Issue by BrutuZ** [croneter/PlexKodiConnect#540]
I noticed PKC is saving the show fanart to its seasons if they are empty\not set on Plex.
This causes an issue when exporting the Kodi Library to backup the metadata, as it saves a duplicate of the show fanart for each season. Kodi saves `fanart.jpg`, but the exact same file is also saved as `season##-fanart.jpg` for each season.

This is probably caused by the fact that Plex always sets the `art` property of seasons, but uses the same path of the series' fanart if no image was set for it.
Could PKC check if a season's `art` value is the same as the one on its `parentKey`, and only set it on Kodi if false?

For quick reference:
 - Season XML:
```
<Directory (...) parentKey="/library/metadata/239242" (...) art="/library/metadata/239242/art/1538223807">
```
 - Show XML:
```
<Directory key="/library/metadata/239242/children" (...) art="/library/metadata/239242/art/1538223807">
```

**Comment by croneter** [comment]
How is that an issue, exactly?

**Comment by BrutuZ** [comment]
Incorrect data being written to the database. Seasons are getting metadata they shouldn't have in Kodi

**Comment by croneter** [comment]
But what is the actual drawback for you as a Kodi user?

**Comment by BrutuZ** [comment]
It has the minor nuisance of making every season have the same fanart backdrop when navigating inside Kodi, ignoring the show's extrafanarts.
But mainly the resulting files of a Kodi Library Export. Note the 16 extra copies of `fanart.jpg` as `seasonXX-fanart.jpg`:
![Multiple copies](https://i.imgur.com/oljuUnr.png)

**Comment by croneter** [comment]
I will have a look. But do tell me - why would you backup Kodi when everything is on your Plex Server?!

**Comment by BrutuZ** [comment]
Had a bad time rearranging libraries in Plex, plenty of customization lost because you have to create the libraries from scratch and deal with parsing all the data again. Having data backed up this way the "Local Metadata Agent" is a huge help.
It also helps keeping the library organized for a Kodi-only device used mostly when I have internet issues, something else Plex do not play very well with.

**Comment by croneter** [comment]
I still don't understand - you'd need to look for solutions on the PLEX side, not Kodi, if you are using PKC. The PMS is the source of all your metadata and most artwork...

E. G. If you have to use one of your Kodi backups, plex could not care less. PKC will even overwrite the Kodi side of things with everything "Plex"

**Comment by BrutuZ** [comment]
Plex will use the local images when "Local Metadata Agent" is enabled, and if I want the .nfo metadata too, there are agents like "XBMC .nfo Importer" to parse them :)

The thing is, Plex is great on fetching the metadata online, but when it comes to manipulating it, it's a nightmare. That includes using the same info for the same files on multiple libraries. If Plex gets the metadata right on the first try, then all should be fine (if you don't mind waiting for it to guess and download everything again), but if there's a mistake or you customized a image, tag, or some other info, it will not be carried away. That's where the local metadata backup comes in handy, which you can't do with Plex.

**Comment by croneter** [comment]
Implemented in PKC 2.5.9
